qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/QuteLanguageServer.java

```diff
@@ -89,7 +88,7 @@
  */
 public class QuteLanguageServer implements LanguageServer, ProcessLanguageServer, QuteLanguageServerAPI,
 		QuteProjectInfoProvider, QuteJavaTypesProvider, QuteResolvedJavaTypeProvider, QuteJavaDefinitionProvider,
-		QuteDataModelProjectProvider, QuteUserTagProvider, QuteTemplateJavaTextEditProvider, QuteJavadocProvider,

```

Comment:
Any reason why this interface is no longer needed?

Reply:
It was an error, QuteTemplateJavaTextEditProvider is a method which must b eimplemented on client side and not on server side. As QuteTemplateJavaTextEditProvider has been implemented with default, we don't see this problem.

It is again a good point to remove default implementation.